### ISSUE TITLE
Measure repeat counts should have text style and default to something smaller

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#13605]
**Describe the bug**
If you enable counts for measure repeats in Format / Style / Measure repeats, the results are too large and the font is not customizable.

**To Reproduce**
1. Set the "Number consecutive measure repeats" option in Format / Style / Measure Repeats
2. Add several consecutive measure repeats somewhere in your score
3. See error: numbers are rendered with the same symbols at the same size as multimeasure rests

**Expected behavior**
A default font that isn't easily mistaken for a multimeasure rest, and the ability to customize it

**Screenshots**
![image](https://user-images.githubusercontent.com/1799936/193705313-116ff29c-3f80-4786-9e7c-dbdfeda62aa0.png)

**Platform information**
 - OS: Linux

**Additional context**
Probably we should have a new text style for this, but meanwhile, we should consider simply a smaller size of the current font, or even just using Edwin.

**Comment by GabeS573** [comment]
In my experience, measure repeats numbers are usually written in a normal serif or sans-serif font. Those multimeasure rest-style numbers sometimes denote repeating multiple bars (2-bar repeat, 4-bar repeat, etc.) which could cause confusion.

Examples:
![image](https://user-images.githubusercontent.com/91428485/194097124-0657c01f-4059-4143-9744-7eb9677e7b32.png)
Repeat Count

![image](https://user-images.githubusercontent.com/91428485/194097289-8d5d0e2f-5958-48f2-92a6-27be10bbd4e8.png)
2-bar repeat

**Comment by RobFog** [comment]
One for @oktophonie to look at?

**Comment by rolodoom** [comment]
Does anyone has info on this topic? As 2026 is still not possilbe to change the font style for this numbers!!!

**Comment by mnorrisvuw** [comment]
Agree, this is a no-brainer — needs its own text style definition.

**Comment by jeetee** [comment]
@rolodoom  All the updates and info are listed right above your question. Any future updates will appear here as well. The current status thus remains unchanged: the issue is open for any contributor to contribute this new feature.

@mnorrisvuw  In the future, you can "thumbs up" the original starting post to let the devs (and community) be aware that you too would like to see this implemented. There is no need for an additional "me too" post, so the topic can remain as clean as possible for those willing to pick it up in the future.
